test/web.test.js

```diff
@@ -16,32 +16,30 @@ test('Web resources belong to a single web', function(t) {
 
   var w1 = nap.web()
     , w2 = nap.web()
-    , fn = function() {}
+    , fn = function() { t.ok(true, 'resource added to w1') }
```

Comment:
This is the result of the change, as the handler is not accessible directly, but will be called in `wrapped`.